docs/pages/dev/reference/API/org_unit_registry.md

```diff
@@ -0,0 +1,423 @@
+# Prerequisite
+
+## Possible fields
+- `parent_id`
+- `name`
+- `opening_status`
+- `org_unit_type_id`
+- `groups`
+- `geo_point`
+- `shape`
+- `instances`
+
+# POST /api/orgunits/changes/?app_id=..
+
+API used to create or modify an OrgUnit with validation (see feature flags)
+When creating an OrgUnit, the client doesn't know the ID or the OrgUnit was not created yet and we might want to apply 
+some changes still. Therefore, the IDs can be either the DB id or the UUID generated by the client.
+Changes that are not provided are considered not being part of the change request.
+
+## Permissions
+
+User must be authenticated.
+
+## Query parameters
+
+- app_id: String - Must be provided, project for which this is created.
+
+## Body
+
+```json
+{
+  "uuid": "UUID - Client generated UUID",
+  "org_unit_id": "String - id or UUID of the OrgUnit to change, if UUID doesn't exist in DB, it's a new OrgUnit.",
+  "created_at": "Timestamp - format same as /api/instances",
+  "updated_at": "Timestamp - format same as /api/instances",
+  "changes": { "": "Changes is not null and must contain at least one field",
+    "parent_id": "String? - id or UUID of the parent OrgUnit, may be null or omitted.",
+    "name": "String? - Name of the OrgUnit, may be null or omitted.",
+    "opening_status": "Enum<Status>? - One of `open` or `close`",
+    "org_unit_type_id": "Int? - id of the OrgUnitType, may be null or omitted",
+    "groups": "Array of Group ids? - can be empty, null or omitted. Empty means we want to remove all values",
+    "geo_point": { "": "New geopoint for the OrgUnit, may be null or omitted",
+      "latitude": "Double - New latitude of the OrgUnit",
+      "longitude": "Double - New longitude of the OrgUnit",
+      "altitude": "Double - New altitude of the OrgUnit",
+      "accuracy": "Double - New accuracy of the OrgUnit"
+    },
+    "geo-shape": "String? - GeoJSON representation, may be null or omitted",
```

Comment:
After another look at whimsical, it looks like the `geo-shape` cannot be updated:

![map](https://github.com/BLSQ/iaso/assets/281139/c25bfce6-0c04-4596-a07d-cc6c9d15852c)

The user can signal an outdated map:

> You cannot edit a shape. Instead, you can raise the fact that this shape is outdated.
> Do you confirm the shape is outdated and needs an update?

Therefore we may need a boolean here instead of a GeoJSON representation.

Reply:
My understanding is that we cannot edit a shape because we don't have a proper tool to do it correctly. Still, it should be possible, I guess.
But we can add a boolean on top of it.

Reply:
```suggestion
```

We won't implement `geo-shape`.